homeassistant/components/tasmota/sensor.py

```diff
@@ -99,11 +109,13 @@
     SENSOR_PROXIMITY: {ICON: "mdi:ruler"},
     SENSOR_REACTIVE_POWERUSAGE: {DEVICE_CLASS: DEVICE_CLASS_POWER},
     SENSOR_STATUS_SIGNAL: {DEVICE_CLASS: DEVICE_CLASS_SIGNAL_STRENGTH},
+    SENSOR_STATUS_RSSI: {DEVICE_CLASS: DEVICE_CLASS_SIGNAL_STRENGTH},
```

Comment:
DEVICE_CLASS_SIGNAL_STRENGTH expects the unit to be `dB` or `dBm`, I think we should just set an icon instead for this sensor.

Reply:
Right. Fixed!